Modules/SubModule_ASL/xASL_wrp_RealignASL.m

```diff
@@ -334,96 +343,94 @@ function xASL_wrp_RealignASL(x,bSubtraction)
 end
 
 %only run if ENABLE is on
-if bExecutedENABLE == 1;
-    if bSubtraction && nFrames>10 % if we performed outlier exclusion
-        tValue(1:3) = tValue(4); % for nicer plotting
+if bENABLE && bSubtraction && nFrames>10 % if we performed outlier exclusion

```

Comment:
What happened here? Please don't only add a tab, unless it is in a separate commit (I didn't check that). But now I cannot see what the difference is, so I hope the difference is only a tab indentation ;)